sdk/include/opentelemetry/sdk/trace/simple_processor.h

```diff
@@ -27,14 +34,16 @@ class SimpleSpanProcessor : public SpanProcessor
 
   std::unique_ptr<Recordable> MakeRecordable() noexcept override
   {
-    return exporter_->MakeRecordable();
+    auto result = exporter_->MakeRecordable();
+    return result;
```

Comment:
I think we can collapse this into one line again.

Reply:
DOH!  Good catch :)